app/views/shared/_subnavigation.html.erb

```diff
@@ -52,12 +54,17 @@
                                   title: t("shared.go_to_page") + t("layouts.header.budgets") %>
       </li>
     <% end %>
-    <li>
-      <%= link_to t("layouts.header.help"),
-                    help_path,
-                    accesskey: "6",
-                    class: ("is-active" if current_page?(help_path)),
-                    title: t("shared.go_to_page") + t("layouts.header.help") %>
-    </li>
+
+    <% unless content_block("main_navigation_left", I18n.locale) || content_block("main_navigation_right", I18n.locale) %>
```

Comment:
Hi @ziyan-junaideen,

I don't understand why hide "Help" menu if there are a `content_block main_navigation_left or content_block main_navigation_right` 🤔

Reply:
I understand. I guess Devin is the best to answer the question. @devinbalkind ^^^

I think he is not part of the team. Will ping him on Slack.

Reply:
Is there a better way to hide "Help" than doing it through this method?

Reply:
I think is better remove this condition for now. If any fork wants to remove this link can create a custom view on `/views/custom/shared/_subnavigation.html.erb` and remove it. ☺️ 

Anyway, I take note of this to create a setting to "enable/disable" Help page and can use something like `<% if feature?(:help_page) %>` how the rest of the subnavigation links 😌

Reply:
Sounds good to me. Thank you.